stock_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2019 Eficent Business and IT Consulting Services S.L.
+# © 2019 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Stock with Operating Units",
+    "summary": "Adds the concept of operating unit (OU) in stock management",
+    "version": "13.0.1.0.0",
+    "category": "Generic Modules/Sales & Purchases",
+    "author": "Eficent, "
+    "Serpent Consulting Services Pvt. Ltd., "
+    "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/operating-unit",
+    "depends": ["stock", "account_operating_unit"],
```

Comment:
```suggestion
    "depends": ["stock", "operating_unit"],
```
IMO, I think `operating_unit` is enough. What do you think about it?

Reply:
I think so too. Updated it according to your comment. Thanks!